src/app/components/common/common.css

```diff
@@ -49,6 +49,10 @@
         pointer-events: none;
     }
 
+    .p-disabled-overlay{
+        cursor : not-allowed !important;
```

Comment:
We should avoid using  !important. It would be better to use CSS Specificity .

Reply:
We cannot add `!important` to the component styles. Could you please remove and update the code?